src/main/java/com/ldtteam/structurize/client/gui/WindowSwitchPack.java

```diff
@@ -88,6 +98,12 @@ public WindowSwitchPack(final Supplier<BOWindow> prevWindow,
 
         this.prevWindow = prevWindow;
         this.packPredicate = packPredicate;
+
+        findPaneOfTypeByID(FILTER_NAME, TextField.class).setHandler(input -> {
+            filter = findPaneOfTypeByID(FILTER_NAME, TextField.class).getText();
```

Comment:
well, you can use input variable, and you can lowercase once

_we really should introduce delayed application so we dont trigger filter 10 times per second_